### ISSUE TITLE
The ability to manipulate numbers using a textbox in the control bindings for joysticks/controllers as well as the current slider

### ISSUE THREAD

**Issue by anth1256** [FloatingOriginStudios/BalsaIssues#47]
![20210901115845_1](https://user-images.githubusercontent.com/74987984/131590898-46a701b8-d74f-4a50-86fe-f1aee42a4a21.jpg)

I find it hard to get the exact number I want by using the sliders at times.

Not a major concern but when I want a specific number it can be fiddly to get it right.

**Comment by Moach** [comment]
Yes, this is a know little nuisance.  I had thought of an alternative which would be that a right-click should snap the slider to the nearest round number.  I think that would be easier to implement and should have the same convenience as a result

**Comment by anth1256** [comment]
Nearest 1 number like 0.10 or 1.0 depending on where it is you mean?

So if it was 1.31 it would drop to 1.3? or 0.32 would drop to 0.30? sounds good.
